src/swap/changelly.js

```diff
@@ -169,11 +183,13 @@ export function makeChangellyPlugin(
       userSettings: Object | void,
       opts: { promoCode?: string }
     ): Promise<EdgeSwapQuote> {
+      log.warn(`${pluginId} swap requested ${JSON.stringify(request)}`)
```

Comment:
Again, we are going to get a ton of redundant logs this way. The core should simply log this exactly once before even calling the plugins.